src/admin/python/lsst/qserv/admin/qservCli/launch.py

```diff
@@ -335,7 +336,7 @@ def mypy(
         "--exclude",
         "testing",
     ]
-    mypy_ini_file = os.path.join(qserv_build_root.format(user=user), "mypy.ini")
+    mypy_ini_file = os.path.join(qserv_build_root.format(user=user), mypy_cfg_file)
```

Comment:
Besides "pleasing the boss" (which is probably a good justification by itself 😹 ), what's a reason for changing an expectation from finding this file in this root folder of the build tree to its sub-folder `src/admin/python/mypy.ini`? In the older version of the code, the file was expected to be found at:
```
<QSERV_BUILD_ROOT>/mypy.ini
```
And now it's expected to be at:
```
<QSERV_BUILD_ROOT>/src/admin/python/mypy.ini
```
Is the source tree still available within the binary container?

Reply:
it was and will be available in the build container (when qserv sources are mounted in). It never was and is not part of the run image.
The reason is to keep the top-level directory of the qserv sources free of clutter, it's a style preference on @fritzm 's part.